<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hello world GUI Application</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="19" column="5" colspan="2">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="15" column="5">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="16" column="5" colspan="2">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Total amount of likes:</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="18" column="5" colspan="2">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Total amount of videos:</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="17" column="5" colspan="2">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
     </widget>
    </item>
    <item row="13" column="5" colspan="2">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
    </item>
    <item row="14" column="5" colspan="2">
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>Remove</string>
      </property>
     </widget>
    </item>
    <item row="13" column="0" rowspan="7" colspan="5">
     <widget class="QTableView" name="tableView"/>
    </item>
    <item row="1" column="0" rowspan="9" colspan="7">
     <widget class="QListView" name="listView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>506</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionLoad_data"/>
    <addaction name="actionSave_data_as_a_file"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionLoad_data">
   <property name="text">
    <string>Load data from a file</string>
   </property>
  </action>
  <action name="actionSave_data_as_a_file">
   <property name="text">
    <string>Save data as a file</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onRemoveButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>413</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>565</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>label</sender>
   <signal>linkActivated(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>on_tableview_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>385</x>
     <y>235</y>
    </hint>
    <hint type="destinationlabel">
     <x>569</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onLoadButtonPushed()</slot>
  <slot>onRemoveButtonClicked()</slot>
  <slot>onAddButtonClicked()</slot>
  <slot>on_tableview_clicked()</slot>
  <slot>onSaveButtonClicked()</slot>
 </slots>
</ui>
